packages/edit-site/src/components/site-hub/index.js

```diff
@@ -149,8 +146,7 @@ export const SiteHubMobile = memo(
 						) }
 					>
 						<Button
-							// TODO: Switch to `true` (40px size) if possible
-							__next40pxDefaultSize={ false }
+							__next40pxDefaultSize
```

Comment:
This can be seen when you visit and check for go to back button on top left corver in site editor mode for mobile mode.

**Note:** It has fixed height, so changing this does not change before and after images.

| Before | After     |
| :-------- | :------- |
| <img width="419" alt="site-hub-mobile-editor-before" src="https://github.com/user-attachments/assets/f1f8f9d3-9178-425a-8e41-4e4047338d85"> | <img width="419" alt="site-hub-mobile-editor-after" src="https://github.com/user-attachments/assets/12c80da0-d228-4c9f-a858-df6e0022b33b"> |